march_simulation/launch/march_world.launch

```diff
@@ -38,6 +38,7 @@
             <arg name="gui" value="$(arg gazebo_ui)" />
             <arg name="paused" value="true"/>
             <arg name="use_sim_time" value="$(arg use_sim_time)"/>
+            <arg name="verbose" value="true" />
```

Comment:
I am not sure whether to always have this enabled. You should create an argument to set this value.

Reply:
I think it is good to set it to true. Without verbose, Gazebo does not print any errors for plugins and that can be frustrating if you think the simulation works and it must be your code, but it really is just the plugin not working.